wordpress-mobile/WordPress-iOS
# UL&S: Re-establishes "signed in" tracking.

WPAuth: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/447

In 15.6 we removed tracking of `signed_in` in some scenarios.  This PR re-establishes tracking `signed_in`.

## To test:

Filter your Xcode console by "Tracked: signed_in".

Test any sign in flow and make sure `signed_in`is tracked when successful.

## PR submission checklist:
